docs/user/ppl/functions/relevance.rst

```diff
@@ -352,6 +352,33 @@ Another example to show how to set custom values for the optional parameters::
     | 1    | The House at Pooh Corner | Alan Alexander Milne |
     +------+--------------------------+----------------------+
 
+
```

Comment:
You need to add a doc for SQL too with an example.

Reply:
There is already an SQL example here: [functions.rst#highlight](https://github.com/opensearch-project/sql/blob/main/docs/user/dql/functions.rst#highlight)